lms/djangoapps/dashboard/sysadmin.py

```diff
@@ -59,9 +60,13 @@ def __init__(self, **kwargs):
         modulestore_type and return msg
         """
 
-        self.def_ms = modulestore()
+        self.def_ms = check_ms = modulestore()
+        # if the modulestore is wrapped by CCX, unwrap it for checking purposes
+        if isinstance(check_ms, CCXModulestoreWrapper):
+            check_ms = check_ms._modulestore
+
         self.is_using_mongo = True
-        if isinstance(self.def_ms, XMLModuleStore):
+        if isinstance(check_ms, XMLModuleStore):
             self.is_using_mongo = False
```

Comment:
It would be nice to convert this over to use `get_modulestore_type`, since that's a cleaner interface for this check.